package.json

```diff
@@ -28,12 +29,12 @@
     "@typescript-eslint/parser": "5.27.1",
     "ava": "4.3.0",
     "bluebird": "3.7.2",
-    "commitlint-circle": "1.0.0",
     "dockerode": "3.3.2",
+    "eslint": "^8.15.0",
     "execa": "5.1.1",
     "freeport": "1.0.5",
     "graphile-build": "4.12.2",
-    "graphql": "16.5.0",
+    "graphql": "15.8.0",
```

Comment:
why is this downgraded?

Reply:
graphile-build constraint: https://github.com/graphile/graphile-engine/blob/v4/package.json#L49